dist/common/scripts/scylla-housekeeping

```diff
@@ -138,12 +138,15 @@ def check_version(ar):
             params = params + "&rtype=" + repo_type
         versions = get_json_from_url(version_url + params)
         latest_version = versions["version"]
-        latest_patch_version = versions["latest_patch_version"]
+        latest_patch_version = versions.get("latest_patch_version")
     except Exception:
         traceln("Unable to retrieve version information")
         return
 
-    if latest_patch_version != latest_version:
+    if latest_patch_version is None:
```

Comment:
what scenario are you handling here?
Can you also add some general note to the PR to give it context?

Reply:
Did you see the commitlog?